contribs/gmf/examples/search.js

```diff
@@ -14,7 +14,7 @@ var app = {};
 
 
 /** @type {!angular.Module} **/
-app.module = angular.module('app', ['gmf']);
+app.module = angular.module('app', ['gmf', 'gettext']);
```

Comment:
See my other comment regarding `gettext` being a dependency of the `gmf` module.